Dockerfile

```diff
@@ -11,3 +11,4 @@ COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator
 COPY manifests/*.yaml /manifests
 COPY manifests/image-references /manifests
 LABEL io.openshift.release.operator true
+ENTRYPOINT ["/usr/bin/cluster-kube-apiserver-operator"]
```

Comment:
this shouldn't be required.  Snip it from your pull.

Reply:
Why? This is how you build tools packaged in containers meant to be run by users. This is not a bash image (which is the entry point currently) but cluster-kube-apiserver-operator image.

```bash
podman run -it --rm --entrypoint=/usr/bin/cluster-kube-apiserver-operator registry.svc.ci.openshift.org/ocp/4.1-2019-04-29-113133:cluster-kube-apiserver-operator recovery-apiserver create
```
vs.
```bash
podman run -it --rm registry.svc.ci.openshift.org/ocp/4.1-2019-04-29-113133:cluster-kube-apiserver-operator recovery-apiserver create
```

Reply:
bootkube is mishandling it :/ which is a bug, but in the interest of time I am going to drop it for now